.buildkite/scripts/pipelines/security_solution_quality_gate/api-integration-tests.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+if [ -z "$1" ]
+  then
+    echo "No target script from the package.json file, is supplied"
+    exit 1
+fi
+
+source .buildkite/scripts/common/util.sh
+.buildkite/scripts/bootstrap.sh
+
+buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
+
+echo "--- Serverless Security Second Quality Gate"
+cd x-pack/test/security_solution_api_integration
+set +e
```

Comment:
This is a large block to use without errors.  Can we use an exit trap to delete the deployment instead?

Reply:
At the moment are not linked anywhere else than Buildkite direct run by a user. So they would not block/break anything yet. We can move this draft work which is running and in the next PR the runner script will fix things.